media/js/libs/fontfaceobserver/fontfaceobserver-standalone.js

```diff
@@ -0,0 +1,41 @@
+/*
+
+https://github.com/bramstein/fontfaceobserver
+v1.4.5
+
+Copyright (c) 2014 - Bram Stein
```

Comment:
The license is compatible. If you add a copy of the full license text to the top of the file, that will be fine.